.github/workflows/analyze.yml

```diff
@@ -0,0 +1,84 @@
+# Copyright (C) 2025 The Android Open Source Project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Perfetto CI
+on:
+  # 1. continuous
+  schedule:
+    # Run every 6 hours
+    - cron: "0 */6 * * *"
+
+  # 2. postsubmit
+  push:
+    branches:
+      - main
+      - test
```

Comment:
Dow we want this

Reply:
Ah fair point. I was using it only for testing :) removed